pkg/tfbridge/tokens/known_modules.go

```diff
@@ -51,11 +51,11 @@ func knownModules[T info.Resource | info.DataSource](
 				break
 			}
 		}
-		var err error
 		if mod == "" {
-			err = fmt.Errorf("could not find a module that prefixes '%s' in '%#v'", tk, modules)
+			return apply("", upperCamelCase(tk), elem,
+				fmt.Errorf("could not find a module that prefixes '%s' in '%#v'", tk, modules))
 		}
-		return apply(moduleTransform(mod), upperCamelCase(strings.TrimPrefix(tk, mod)), elem, err)
+		return apply(moduleTransform(mod), upperCamelCase(strings.TrimPrefix(tk, mod)), elem, nil)
```

Comment:
Can we revert these changes after https://github.com/pulumi/pulumi-terraform-bridge/pull/2882/commits/ccb879b5c86122001f4ee31121f497bf93d07475?